cli/azd/pkg/project/scaffold_gen.go

```diff
@@ -474,7 +492,6 @@ func genBicepParamsFromEnvSubst(
 			result += "}"
 			previous = loc.stop + 1
 		}
-		result += "'"
```

Comment:
1. For plain text, I remoted quote, so it can apply to my new code.
2. For variable, I added `${` and `}` to make it can be evaluated in quote.